mmv1/third_party/terraform/tests/resource_google_project_test.go

```diff
@@ -13,13 +13,6 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-	"google.golang.org/api/cloudresourcemanager/v1"
-)
-
-var (
```

Comment:
Can we make this change in a separate PR? It's hard to follow where these values went in this one.

Reply:
Sure. That makes sense.

Reply:
For pname in a separate PR we could also just eliminate it altogether and use the project id across-the-board.